source/configuring_domain/02_managing_legal_information/legals.rst

```diff
@@ -68,22 +68,68 @@ To specify the Terms and Conditions and the Privacy Policy of a portal, go to Co
 Licenses
 --------
 
-When configuring the metadata of a dataset in the Information tab, the license attached to that dataset should be defined, using a drop-down selection of predefined licenses. To define the licenses listed in the drop-down selection, go to Configuration > Legals. In this subsection of the back office menu, look for the "Licenses" area.
+When configuring the metadata for a dataset in the Information tab, it is important to define the license that specifies how others can use the data.
 
-.. image:: images/license_configuration.png
+Configure licenses
+~~~~~~~~~~~~~~~~~~
 
-1. Click the Add license button.
-2. In the Labels column, write the label of the license. The label will be displayed both in the drop-down selection in the back office, and in the Information tab of the published dataset in the front office. There must be one label per language available on the domain.
```

Comment:
is this part about labels (mainly where it will appear + one mandatory label per language) gone in the new version?